app/views/layouts/application.html.erb

```diff
@@ -34,7 +34,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title><%= yield :title %> - GOV.UK</title>
```

Comment:
Not convinced this needs to be ported as GOV.UK _can't_ be translated?

Reply:
Yes, I think that's probably true! I don't think I'll leave it as a hard coded string though, just because that would involve adding this file to the exclude list for the linter rules, or adding a single line exception. The former would mean other strings in this file would be missed and the latter makes the file more complicated/ugly. There's several examples of strings that are not currently translated (their corresponding keys in non-`en` locale files have nil values).